Keas.Mvc/ClientApp/components/Equipment/EquipmentEditValues.tsx

```diff
@@ -79,7 +98,9 @@ export default class EquipmentEditValues extends React.Component<IProps, {}> {
                             }
                             onChange={e => this.props.changeProperty("make", e.target.value)}
                         />
-                    </div>
+                    </div>
+                    )}
+                    {this.props.selectedEquipment.type !== "Card" && (
```

Comment:
"in the future" this would be better not inline w/ markup.  Maybe a method like `shouldShowForType(type, prop)` would be good, and that method would return true/false w/o having to change multiple places inline.